Microsoft.Research/Contracts/WindowsBase/System.ComponentModel.ICollectionView.cs

```diff
@@ -0,0 +1,504 @@
+﻿using System;
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+
+namespace System.ComponentModel
+{
+#if !SILVERLIGHT && !NETFRAMEWORK_3_5 // => types defined in System.Windows!
+
+    // Summary:
+    //     Provides information for the System.ComponentModel.ICollectionView.CurrentChanging
+    //     event.
+    public class CurrentChangingEventArgs : EventArgs
```

Comment:
:bulb: I would prefer to see this in its own file. This isn't a big deal for now; it will likely be addressed automatically as part of #63 or a subsequent related PR.